### DISCUSSION TITLE
IsarError: SchemaMismatch: The schema of the existing instance does not match.

### DISCUSSION THREAD

**Discussion by ghost** [isar/isar#327] [Help] [answered]
Hello,

afaik, according to the [docs](https://isar.dev/schema.html#schema-migration), adding a field is allowed. But I am not sure why this is still happening.

![Screenshot 2022-04-06 at 22 10 21](https://user-images.githubusercontent.com/8101584/162062410-58433bfd-729b-4f75-baa8-7253ca0e1740.jpg)

**Comment by ghost** [comment, marked as answer]
Ok, have to stop the emulator and rebuild again.

**Reply by simc** [reply]
Yes, unfortunately hot reload / restart doesn't work with schema changes
